pages/challenge.js

```diff
@@ -0,0 +1,146 @@
+              something like this:
+              <br />
+              <br />
+              <img
+                src={ExampleImage}
+                alt="example of GitHub input thus far"
+                className={styles.blockImage}
+              />
+              <br />
+              <br />
+              Once you confirm the similarities, click
+              <img src={ProposeButton} alt="the 'Propose file change' button" width="125px" />
+            </li>
+            <li>
+              You should now be at the &quot;
+              <b>Open a pull request</b>
+              &quot; screen. We do not wish to ask ourselves for permission to merge our new branch
+              into our own fork! Instead,&nbsp; click{' '}
+              <OutboundLink analyticsEventLabel={`${CompareLink}`} href={`${CompareLink}`}>
+                {' '}
+              </OutboundLink>
+              &nbsp;, to open Operation Code&apos;s &quot;New pull request&quot; interface. You
+              should see a &quot;Compare changes&quot; headline. Just below that is a link within
+              the text:&nbsp; &apos;compare across forks&apos; - click it. Now, click on the
+              selector that says &apos; head fork&apos; at the beginning, and choose your fork.
+              Click the next selector to the right, and choose your new branch. Now, you&apos;re
+              comparing Operation Code&apos;s master branch with your new fork&apos;s branch, and
+              you may click
+              <img src={PRButton} alt="the 'Create pull request' button" width="115px" />
+              to create your first Pull Request! We hope you come to love that button...
+            </li>
+            <blockquote>
+              <i>
+                <u>NOTE:</u>
+              </i>{' '}
+              A pull request is how people throughout the world are able to contribute to open
+              source software - like Operation Code&apos;s website! When you submit a pull request
+              it notifies the maintainers of the project, and runs some automated checks. The
+              maintainers then look at the new changes, and decide if they want it merged into their
+              repository.
+            </blockquote>
+            <br />
+            <li>
+              When you&apos;re ready, click the &quot;Create pull request&quot; button. Our staff
+              will be notified and a few minutes after the pull request is accepted and merged your
+              name will show up below!
+            </li>
+          </ol>
+          <h6 className={styles.centerText}>
+            Congratulations - you&apos;ve made your first open source commit!
+          </h6>
+        </Section>
+        <Section title="Your Name Here" className={styles.displayBlock}>
+          <h6 className={styles.centerText}>
+            Here is a list of the people that have completed this before you:
+          </h6>
+          {names.map(name => (
```

Comment:
this seems ripe for an `<ol>` and instead of `<div>` each name should be in an `<li>` you can leave default list stylings